src/core/service.c

```diff
@@ -1440,6 +1440,93 @@ static bool service_exec_needs_notify_socket(Service *s, ExecFlags flags) {
         return s->notify_access != NOTIFY_NONE;
 }
 
+static int service_create_monitor_md_env(Job *j, char **ret) {
+        assert(j);
```

Comment:
`assert(ret)` too, also the asserts below the declarations

Reply:
sorry, fixed.